packages/react-native-bpk-component-text/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-bpk-component-text",
-  "version": "4.2.1",
+  "version": "4.2.2",
```

Comment:
Was this updated manually? If so, can you rever it? This is handled by the release script.

Reply:
No I updated it, ok sounds good